postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -319,15 +323,17 @@ def _filter_and_normalize_statement_rows(self, rows):
         normalized_rows = []
         for row in rows:
             total_count += 1
-            if not row['datname']:
+            if (not row['datname'] or not row['query']) and row.get(
+                'backend_type', 'client backend'
+            ) == 'client backend':
```

Comment:
is it possible for queries with this backend type to fulfill this criteria `(coalesce(TRIM(query), '') != '' AND query_start IS NOT NULL {extra_filters})`? if so, is there a reason we don't want to collect it?

Reply:
oh actually sorry i misread this logic... looks good